management-service/src/main/java/pl/pwr/zpi/cluster/service/ClusterService.java

```diff
@@ -27,9 +29,42 @@ public class ClusterService {
     private final ReceiverService receiverService;
     private final MetadataService metadataService;
 
+    public List<ClusterMetadataDTO> getAllClusters() {
+        List<ClusterMetadataDTO> clusters = metadataService.getAllClusters();
+        clusters.forEach(this::setClusterConfigurationForMetadata);
+        return clusters;
```

Comment:
you can inline this

Reply:
How?

Reply:
return metadataService.getAllClusters().stream()
        .peek(this::setClusterConfigurationForMetadata)
        .toList();

Reply:
Fixed, thanks!